### PR TITLE
update topic replica to 3

### PR DESCRIPTION
**Pull request by ldpliu**
## Summary
As we discussed in meeting, we should update the topic replica to 3.
## Related issue(s)

Fixes #

### CODE REVIEW

File: operator/pkg/transporter/strimzi_transporter.go [hunk 1]
```diff
@@ -261,9 +262,22 @@ func (k *strimziTransporter) CreateTopic(topic *transport.ClusterTopic) error {
...
+			if errors.IsNotFound(err) {
+				if e := k.runtimeClient.Create(k.ctx, k.newKafkaTopic(topicName)); e != nil {
+					return e
+				}
+				return nil
+			}
+			return err
+		}
+		desiredTopic := k.newKafkaTopic(topicName)
+		if !reflect.DeepEqual(desiredTopic.Spec.Replicas, kafkaTopic.Spec.Replicas) {
```
Comment (yanmxa) [inline operator/pkg/transporter/strimzi_transporter.go:275]: If we only assert the replicas, maybe we don't need the `reflect.DeepEqual` method

Comment (ldpliu) [reply operator/pkg/transporter/strimzi_transporter.go]: > If we only assert the replicas, maybe we don't need the `reflect.DeepEqual` method

Changed.

**Review by clyang82 — CHANGES_REQUESTED** [review]
Need to update the document for the storage size

File: operator/pkg/transporter/strimzi_transporter.go [hunk 2]
```diff
@@ -261,9 +261,22 @@ func (k *strimziTransporter) CreateTopic(topic *transport.ClusterTopic) error {
...
+		if err != nil {
+			if errors.IsNotFound(err) {
+				if e := k.runtimeClient.Create(k.ctx, k.newKafkaTopic(topicName)); e != nil {
+					return e
+				}
+				return nil
+			}
+			return err
+		}
+		desiredTopic := k.newKafkaTopic(topicName)
```
Comment (clyang82) [inline operator/pkg/transporter/strimzi_transporter.go:273]: after inner discussion, we agree to reconcile the kafkatopic and kafkauser. in other words, we do not allow the user to modify the kafkatopic and kafkauser. @hchenxa @yahliu any concerns for this behaviour change?

**Review by clyang82 — APPROVED** [review]
LGTM

**Pull request merged**